src/vos/tests/vts_mvcc.c

```diff
@@ -1459,7 +1459,12 @@ uncertainty_check_exec_one(struct io_test_args *arg, int i, int j, bool empty,
 				expected_arc = -DER_EXIST;
 		}
 	}
-	if (is_punch(w) && we > bound)
+	if (is_punch(a) && expected_arc != -DER_NONEXIST && we > ae &&
+	    a->o_wlevel <= w->o_wlevel) {
+		expected_arc = -DER_TX_RESTART;
+	}
+
```

Comment:
No blank line, please? It breaks the scope of the comment on line 1449-1450.

Reply:
fixed